test/verify/check-connection

```diff
@@ -1425,7 +1425,9 @@ server {
         m.execute("systemctl start nginx")
 
         def run_ws(extra_opts=""):
-            m.spawn(f"su -s /bin/sh -c '{self.libexecdir}/cockpit-ws --address=127.0.0.1 --for-tls-proxy {extra_opts}' cockpit-wsinstance", "ws.log")
+            m.spawn(
+                f"runser -u cockpit-wsinstance -- {self.libexecdir}/cockpit-ws "
```

Comment:
Stupid typo, [breaking the test](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20613-67c64c23-20240618-080239-fedora-40-other/log.html#158). Pushing a fix.